internal/service/scaffold/options.go

```diff
@@ -35,7 +35,7 @@ type Options struct {
 	ModuleChannel             string
 }
 
-func (opts Options) validate() error {
+func (opts Options) Validate() error {
```

Comment:
What do you think about keeping this private and adapt the tests to be whitebox tests?

Reply:
What do you mean with whitebox tests? The `Validate()` func is the only API the `Options` struct has. SUT and tests are in different pkgs as well: `scaffold` and `scaffold_test`.